client/my-sites/no-results/index.jsx

```diff
@@ -3,27 +3,25 @@
  */
 import React from 'react';
 
+/**
+ * Internal dependencies
+ */
+import { useTranslate } from 'i18n-calypso';
+
 /**
  * Style dependencies
  */
 import './style.scss';
 
-class NoResults extends React.Component {
-	static defaultProps = {
-		text: 'No results',
-		image: false,
-	};
-
-	render() {
-		return (
-			<div className="no-results">
-				{ this.props.image ? (
-					<img className="no-results__img" src={ this.props.image } alt="" />
-				) : null }
-				<span>{ this.props.text }</span>
-			</div>
-		);
+export default function NoResults( { text, image = false } ) {
```

Comment:
Do we even need the default to `false`? Doesn't seem like it.

Reply:
I don't see the `false` default make any difference. It seems we can drop it.